File: 1550-three-consecutive-odds/1550-three-consecutive-odds.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
const threeConsecutiveOdds = arr => {
  for (let i = 0, x = new Array(3); i < arr.length; i++) {
    x[i % 3] = arr[i] % 2;
    if ('[1,1,1]' === JSON.stringify(x)) return true;
  }
  return false;
};

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int cnt = 0;
        for(int num : arr){
            if(num % 2 ==0){
                cnt = 0;
            }
            else{
                cnt++;
            }
            if(cnt == 3){
                return true;
            }
        }
        return false;
    }
}